### PR TITLE
feat: (MOSSMVP-116) Frontend App header

### PR DESCRIPTION
**Pull request by jevgenis33**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -33,18 +33,6 @@ members = [
...
-# compiling slower, so it's only enabled in release mode.
-lto = false
-opt-level = 3
-panic = "abort"
-codegen-units = 1
-
-[profile.dev.package.sqlx-macros]
-opt-level = 3
-
-
```
Comment (g10z3r) [inline Cargo.toml:47]: There is no need to delete this, these optimizations will be needed when we build the binary, so let's just comment it out for now

File: Cargo.toml [hunk 2]
```diff
@@ -55,7 +43,6 @@ moss_base = { path = "lib/moss_base" }
 
 base = { path = "platform/base" }
 configuration = { path = "platform/configuration" }
-workbench = { path = "platform/workbench" }
```
Comment (g10z3r) [inline Cargo.toml:58]: This module is also needed, maybe I deleted it when changing versions, but we need it

File: package.json [hunk 3]
```diff
@@ -4,6 +4,11 @@
   "scripts": {
     "build": "turbo build",
     "dev": "turbo dev",
+    "desktop": "cd view/desktop && pnpm tauri dev",
+    "sb": "cd view/storybook && pnpm dev",
+    "doki": "cd view/docs && pnpm dev",
+    "web": "cd view/web && pnpm dev",
+    "desktop-web": "cd view/desktop && pnpm vite dev",
```
Comment (g10z3r) [inline package.json:11]: doki?) let's change this to `docs`

File: view/desktop/src/components/types.ts
Comment (g10z3r) [inline view/desktop/src/components/types.ts:1]: Noisy comments... there is obvious code here, there is no need for so many comments

File: view/desktop/src/components/window-controls/libs/plugin-os.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+// A helper function to get the OS type, which returns a Promise
+export function getOsType(): Promise<OsType> {
+  if (!osTypePromise) {
+    // If the module was already loaded, just return the result
+    return Promise.resolve(osType!); // Use non-null assertion
+  }
+
+  // If the module is still loading, wait for it to finish and return the result
+  return osTypePromise;
+}
```
Comment (g10z3r) [inline view/desktop/src/components/window-controls/libs/plugin-os.ts:26]: Noisy comments

**Review by g10z3r — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (jevgenis33) [reply Cargo.toml:47]: Restored and commented out.

File: package.json (hunk 3 above)
Comment (jevgenis33) [reply package.json:11]: **pnpm docs** already exists:)

**Pull request merged**